README.md

```diff
@@ -18,59 +29,59 @@ If you're new here, here are some things you should know:
  - Check the `CONTRIBUTING.md` file for a guide on how to get started
  - This is a 100% volunteer-supported project, please be patient with your correspondence. We do handle issues and PRs with more fervor during Hacktoberfest & Conferences, but most (all?) of us have day jobs and so responses to questions / pending PRs may not be immediate. Please be patient, we'll get to you! :)
 
-Please feel free to join us on Slack! You can sign up at https://rubyforgood.herokuapp.com We're in #human-essentials
+#### 💬 Join us on slack
+Please feel free to join us on Slack!
+You can sign up at https://rubyforgood.herokuapp.com and find us in #human-essentials.
 
-The core team leads are: @albert @angela @sean @scott
 There are numerous other folks that can chime in and answer questions -- please ask and someone will probably be there to help!
 
-# README
-
-## About
+## 🛠️ Getting Started
 
-This application is an inventory management system that is built to address the needs of [Diaper Banks](https://nationaldiaperbanknetwork.org/diaper-need/) as directly and explicitly as possible, and later adapted to meet the need of other Essentials Banks.  Essentials Banks maintain inventory, receive donations and other means of intaking human essentials supplies (e.g. diapers, period supplies), and issue Distributions to community partner organizations. Like any non-profit, they also need to perform reports on this data, and have day-to-day operational information they need as well. This application aims to serve all those needs, as well as facilitate, wherever possible the general operations of the Diaper Bank themselves (eg. through using barcode readers, scale weighing, inventory audits).
+### Install Ruby
+You will need to first install the required ruby version specified in the `.ruby-version` file.  It is highly recommended you use a ruby version manager like [rbenv](https://github.com/rbenv/rbenv) or [asdf](https://asdf-vm.com/)
 
-For a general overview of the application, please see the [Application Overview](https://github.com/rubyforgood/human-essentials/wiki/Application-Overview) wiki article.
+GoRails has a very detail oriented guide for installing ruby on Ubuntu, Windows, and macOSX. You can check out that guide [here](https://gorails.com/setup/osx/12-monterey). **Follow only the Installing Ruby step, as our project setup differs**
 
-### Origins
+Once you've succesfully installed ruby. You may proceed to the next section!
 
+### Install Node
+You must install the version of node specified in `.nvmrc`. It is highly recommended to use [nvm](https://github.com/nvm-sh/nvm).
```

Comment:
I'd try to be more specific (Add the step to run `nvm install` ?)

Reply:
I do like the idea of being more specific but the challenge is keeping it up to date. I found linking to other sources that are up to date is a little more future-proof.

Reply:
My train of thought / main concern here is trying to provide context to a person who has never seen a `.nvmrc` file before. 

I'd add a link to the file in the repo so they can see is just a plain text file specifying the version. The part I'm struggling with is how to explain that you can run `nvm install` in the project folder and it should fetch the specified version or you can manually `run nvm install version`

Reply:
@heyapricot I think you did very well yourself. 'You can run nvm install in the project folder and it should fetch the specified version or you can manually run nvm install version'. That was very clear to me.

I don't know how often the version of '.nvmrc' changes, but I think it might be useful to just write it here. Something like: 
Version 12.14.1 (Last review 2022.6.15)

Reply:
> @heyapricot I think you did very well yourself. 'You can run nvm install in the project folder and it should fetch the specified version or you can manually run nvm install version'. That was very clear to me.

Will be adding! Thanks!

> I don't know how often the version of '.nvmrc' changes, but I think it might be useful to just write it here. Something like:

You are right node hardly changes and it is a relatively quick fix if it goes out of date. I can add that in.